frontend/public/components/factory/table.tsx

```diff
@@ -424,13 +442,39 @@ type ComponentProps = {
   kindObj?: K8sResourceKindReference;
 };
 
+const getActiveColumns = (
+  Header: any,
+  componentProps: ComponentProps,
+  activeColumns: Set<string>,
+) => {
+  let columns = Header(componentProps);
+  if (_.isEmpty(activeColumns) && columns.find((col) => col.additional)) {
```

Comment:
Is the `find` call here needed? It seems like we should leave that out.

Reply:
this is correct, its not needed. removed.